packages/studio/src/components/font-tokens.tsx

```diff
@@ -23,6 +25,14 @@ export default function FontTokens(props: FontTokensProps) {
     setText(event.target.value)
   }
 
+  if (fontTokens.length === 0) {
+    return (
+      <EmptyState title="No Tokens" icon={<TypographyIcon />}>
+        The panda config does not contain any `{token}` tokens
```

Comment:
```suggestion
        The panda config does not contain any font tokens
```

Reply:
no I think we should keep it as is, as it depends on which page is rendering this component
https://github.com/chakra-ui/panda/blob/97e1152524af9a1ccd7a6085599919055e07fb63/packages/studio/src/pages/font-sizes.astro#L12
https://github.com/chakra-ui/panda/blob/97e1152524af9a1ccd7a6085599919055e07fb63/packages/studio/src/pages/font-weights.astro#L12

Reply:
Oh I see. Then the strong interpolation needs to be updated.

`${token}`